### PR TITLE
Modified Intermediate_PregnancyDuringART and intermediate_LatestObs

### PR DESCRIPTION
**Pull request by DennisGibz**

### CODE REVIEW

File: Scripts/REPORTING/ServiceDesk.sql
Comment (Marymary-dev) [inline Scripts/REPORTING/ServiceDesk.sql:1]: @DennisGibz Please change the name from LinelistTicketExport to LinelistServiceDeskRequests

File: Scripts/REPORTING/ServiceDesk.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      ,CAST([SDP Point Person Tagged?]   AS nvarchar(100))As SDPPointPersonTagged
+      ,CAST([User's Facility Organization]   AS nvarchar(100))As UserFacilityOrganization
+      ,CAST([Due Date]   AS nvarchar(100))As DueDate
+      ,CAST([Target Version]   AS nvarchar(100))As TargetVersion
+      ,CAST([Solution]    AS nvarchar(100))As Solution
+      ,CAST([County]    AS nvarchar(100))As County  
+      ,CAST([Issue Type]   AS nvarchar(100))As IssueType
+      ,CAST([Resolution Status]   AS nvarchar(100))As ResolutionStatus
+	  INTO REPORTING.dbo.LinelistTicketExport
+  FROM [ODS].[dbo].[OLE DB Destination]
```
Comment (nobert-mumo) [inline Scripts/REPORTING/ServiceDesk.sql:28]: @DennisGibz maybe you can improve on the naming of the ODS table to be more descriptive. Also I thought we should have a fact for this to reading in the reporting table ?

**Review by nobert-mumo — CHANGES_REQUESTED** [review]

**Comment by DennisGibz** [comment]
@Marymary-dev / @nobert-mumo  the naming should now be okay. The report name remains as REPORTING.dbo.Linelist_ServiceDeskRequests

**Comment by nobert-mumo** [comment]
> @Marymary-dev / @nobert-mumo the naming should now be okay. The report name remains as REPORTING.dbo.Linelist_ServiceDeskRequests

@DennisGibz great. What about the reading from ODS. I feel we need a fact for this.

**Review by Marymary-dev — APPROVED** [review]

**Comment by nobert-mumo** [comment]
> > > @Marymary-dev / @nobert-mumo the naming should now be okay. The report name remains as REPORTING.dbo.Linelist_ServiceDeskRequests
> > 
> > 
> > @DennisGibz great. What about the reading from ODS. I feel we need a fact for this.
> 
> @nobert-mumo I agree , we can plan to have this developed

sure, let's just do it right as part of this PR.

**Comment by DennisGibz** [comment]
@nobert-mumo "we can plan to have this developed" I am not getting this.

**Comment by DennisGibz** [comment]
Ah. I now get it. You can help in this? The plan was we first automate what was there since it was very manual.

**Comment by DennisGibz** [comment]
The PR is attached to other two changes which are a bit urgent. Maybe we can have this approved for the ETL purposes then work on the fact in a different workstream.

**Comment by Marymary-dev** [comment]
> The PR is attached to other two changes which are a bit urgent. Maybe we can have this approved for the ETL purposes then work on the fact in a different workstream.

@DennisGibz Am sorry, my bad what @nobert-mumo  says is that we develop the Fact for service desk so that we can have a linelistforServiceDesk and I agree, I thought he was referring to Fact for Pregnant on ART

**Review by Marymary-dev — CHANGES_REQUESTED** [review]
Please develop the model for the Service desk so that we can build up on the Fact and subsequently the linelist

**Pull request closed without merging**